storage_client.go

```diff
@@ -226,23 +226,6 @@ func (rsu *remoteStorageUtil) downloadOneFile(meta *fileMetadata) error {
 				meta.dstFileSize = fi.Size()
 			}
 			return nil
-		} else if meta.resStatus == renewPresignedURL {
```

Comment:
This part of the code is unreachable. Since either an error will be returned on line 206 or it will be nil error and status downloaded.